src/libxtp/qmmolecule.cc

```diff
@@ -16,33 +16,58 @@
  *
  */
 
-#include <votca/csg/pdbwriter.h>
-#include <votca/csg/xyzreader.h>
-#include <votca/csg/xyzwriter.h>
+#include <votca/csg/io/pdbwriter.h>
+#include <votca/csg/io/xyzreader.h>
+#include <votca/csg/io/xyzwriter.h>
+#include <votca/csg/csgtopology.h>
 #include <votca/tools/elements.h>
 #include <votca/xtp/checkpointreader.h>
 #include <votca/xtp/checkpointwriter.h>
 #include <votca/xtp/qmmolecule.h>
+#include "../../include/votca/xtp/fileadapter.h"
 
 using namespace std;
 using namespace votca::tools;
 
 namespace votca {
 namespace xtp {
 
+const tools::DistanceUnit QMMolecule::distance_unit =                             
+    tools::DistanceUnit::angstroms;                                             
+const tools::MassUnit QMMolecule::mass_unit =                                     
```

Comment:
why do you have to define it for every class and not simply generic for votca xtp?

Reply:
This was a design decision I had two options, if I define it in every class then whenever I pass in the class to the unit converter by default it knows how to convert it, I don't have to worry about passing in extra options to specify the units and possibly messing it up. Plus if the units are declared static const there should only be one instantiation per class so it should not increase the memory (I don't know if that was done for the above case will have to look). 

What do you think? 

Maybe declare all the default units used in xtp and package them in a single struct and just include the struct as a static const member variable of the classes/ or equivalent. I don't think it should be a performance problem if done correctly because it should be possible to evaluate the units at compile time.